unwind-pid.c

```diff
@@ -16,6 +17,8 @@ int main(int argc, char **argv)
 	pid_t pid = atoi(argv[1]);
 	if (ptrace(PTRACE_ATTACH, pid, 0, 0) != 0)
 		die("ERROR: cannot attach to %d\n", pid);
+	int status;
+	assert(waitpid(pid, &status, 0) == 0);
```

Comment:
Thanks for the patch. Adding waitpid() here looks very sensible.

However I don't understand why we reorder the header include order and use assert based error checking instead of the if/die approach from the line above.